.github/workflows/test-metal3-helm-chart.yml

```diff
@@ -0,0 +1,111 @@
+name: Test Metal3 Helm Chart
+
+on:
+  pull_request:
+    paths:
+      - '**/assets/metal3/metal3-0.2.0.tgz'
+
+jobs:
+  test-metal3-helm-chart:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+
+    - name: Install K3s
+      run: |
+        curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="server --cluster-init --write-kubeconfig-mode=644 --disable=servicelb" K3S_TOKEN=foobar sh -
+
+    - name: Install Helm
+      run: |
+        curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+        chmod +x get_helm.sh
+        ./get_helm.sh
+
+    - name: Install CertManager
+      run: |
+        export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
+        helm repo add jetstack https://charts.jetstack.io
+        helm repo update
+        helm install cert-manager jetstack/cert-manager \
+          --namespace cert-manager \
+          --create-namespace \
+          --set installCRDs=true \
+          --version v1.11.1 
+
+    - name: Extract Metal3 Helm Chart
+      id: get-chart
+      run: |
+        mkdir helm-metal3-charts
+        tar xvzf assets/metal3/metal3-0.2.0.tgz -C helm-metal3-charts/
```

Comment:
I'm wondering if there's a way to consume the file detected in on/pull_request with a glob, so this will continue to work when we bump the version in future?

I don't see a way to do that in the native GH API but found https://github.com/marketplace/actions/changed-files which might help?

Reply:
Hmm, I can try it but what will be the logic here? 
If it looks for changes in assets/metal3/... and there is more than one change there (if we change things in the previous version as we did in my previous PR), then should it check both versions or just the latest one?

Reply:
Addressed